frontend/src/Notifications.jsx

```diff
@@ -71,44 +122,51 @@ const Notifications = (props) => {
     setAnchorEl(null);
   };
 
-  const { lessonStore } = props;
-
-  const constCheckNotifications = (lessonAttempts) => {
-    lessonStore.lessonAttempts.map((la) => {});
+  const onClickNotification = (notification) => {
+    if (notification.category === 'classes') {
+      notificationStore.deleteNotification(notification.index);
+      history.push(`/dashboard/${notification.category}/${classes[0].id}`);
+    }
   };
 
-  return (
-    <Grid container xs={1} md={1} className={classes.topGrid}>
-      <IconButton color="inherit" onClick={handleClick}>
-        <NotificationsIcon />
-      </IconButton>
+  if (loading || !classes) {
+    return <CircularProgress />;
+  }
 
+  return (
+    <Grid container xs={3} md={3} className={styleClasses.topGrid}>
+      <Grid item xs={3} md={3}>
+        <IconButton color="inherit" onClick={handleClick}>
+          <Badge
+            badgeContent={notificationStore.notifications.length}
+            color="secondary"
+          >
+            <NotificationsIcon />
+          </Badge>
+        </IconButton>
+      </Grid>
       <StyledMenu
         anchorEl={anchorEl}
         keepMounted
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        <StyledMenuItem>
-          <ListItemIcon>
-            <SendIcon fontSize="small" />
-          </ListItemIcon>
-          <ListItemText primary="Sent mail" />
-        </StyledMenuItem>
-        <StyledMenuItem>
-          <ListItemIcon>
-            <DraftsIcon fontSize="small" />
-          </ListItemIcon>
-          <ListItemText primary="Drafts" />
-        </StyledMenuItem>
-        <StyledMenuItem>
-          <ListItemIcon>
-            <InboxIcon fontSize="small" />
-          </ListItemIcon>
-          <ListItemText primary="Inbox" />
-        </StyledMenuItem>
+        {notificationStore.notifications.length > 0 ? (
```

Comment:
Je kan dit het beste schrijven als `(notificationStore.notifications || []).length` (omdat het mogelijk is dat notifications in notificationStore bijvoorbeeld niet bestaat en dat je dan een lege array wilt gebruiken als check.

Reply:
de default in the store is altijd een lege array, so should always be fine

Reply:
Ohja ids heb t nu alsnkg genoteerd zoals Dennis had gezegd maar kan ik nog terug veranderen